styleguide/_layouts/default.jade

```diff
@@ -95,5 +95,6 @@ html(lang="en")
   script.
     $(".imacBanner-headline").fitText(1.1, { maxFontSize: '45px' });
   script.
-    $(".refresh-headline").fitText(0.75, { maxFontSize: '55px' });
-    $(".refresh-headline").not('.refresh-headlineHomepage').fitText(0.7, { maxFontSize: '55px' });
+    $(".refresh-headlineHomepage").fitText(0.8, { minFontSize: '25px', maxFontSize: '55px' });
+    $(".refresh-headlineSecond").fitText(0.75, { minFontSize: '25px', maxFontSize: '55px' });
+    $(".refresh-headlineInterior").fitText(0.8, { minFontSize: '25px', maxFontSize: '55px' });
```

Comment:
I thought you were removing banner changes from this PR?  Requiring new classes for banners and trying to conditionally load javascript is gonna require additional www work. 

This PR should literally only be the bare minimal needed for adding the nascar river.

Reply:
This PR had to also adjust for styling when "putting it all together" as we discussed previously. It's why we waiting on this one until we merged everything else. I removed the changes to add a new way to do banners. I still had to make style adjustments to make the banners match the mock.